src/main/java/com/revature/aspects/LoggingAspect.java

```diff
@@ -66,8 +67,6 @@ public Object logControllers(ProceedingJoinPoint jp) throws Throwable {
 			String controlLog = jp.getTarget() + " invoked " + jp.getSignature() + " throwing: " + e;
 			loggerService.getException().warn(controlLog, e);
 			response.setStatus(500);
-			//TODO comment out throw e on production to block stack trace.
-			throw e;
```

Comment:
It looks like now we are catching and resolving all Throwables that ever occur on the backend.